README.md

```diff
@@ -1,3 +1,7 @@
+[Embedded Viewer](https://replayweb.page/docs/embedding/) Usage Stats: [![](https://data.jsdelivr.com/v1/package/npm/replaywebpage/badge)](https://www.jsdelivr.com/package/npm/replaywebpage)
```

Comment:
Would revert moving this to the top, the h1 should generally be the first thing in the document.